app/Dockerfile

```
@@ -2,29 +2,57 @@
 # https://hub.docker.com/_/python
 
 # The build stage that will be used to deploy to the various environments
-# needs to be called `release` in order to integrate with the repo's
-# top-level Makefile
-FROM python:3-slim AS release
-
-# Install poetry, the package manager.
-# https://python-poetry.org
-RUN pip install poetry
+# This is what the other stages will call as their base
+FROM python:3.11-slim-bullseye AS build
```

nit: since this stage doesn't really build anything (it only installs same base tools like poetry and apt-get stuff), can we name this something else, like `build-tools` or `base` or something
```suggestion
FROM python:3.11-slim-bullseye AS build-tools
```

---

I like `base` best, changing to that in the next commit